src/shared/extensions/ssh.ts

```diff
@@ -123,9 +123,10 @@ export async function startVscodeRemote(
     ProcessClass: typeof ChildProcess,
     hostname: string,
     targetDirectory: string,
-    vscPath: string
+    vscPath: string,
+    user?: string
 ): Promise<void> {
-    const workspaceUri = `vscode-remote://ssh-remote+${hostname}${targetDirectory}`
+    const workspaceUri = `vscode-remote://ssh-remote+${`${user}@` ?? ''}${hostname}${targetDirectory}`
```

Comment:
Does this cause any issues with CodeCatalyst? Our e2e tests wouldn't cover this.

If you haven't tested this yet I can give it a try.

Reply:
I tested it, it doesn't seem to cause any issues with CodeCatalyst. I was still able to open dev environments in the sam way as before. Is there any other key functionality I should be testing here?

Reply:
> Is there any other key functionality I should be testing here?

No, as long as you can connect in a new window then it should be good.